sdk/iot/inc/az_iot_hub_sas.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_span.h>
+#include <az_result.h>
+#include <time.h>
+#include "az_iot_mqtt.h"

```

Comment:
We never use quote-style includes in public or internal header files, only in the private headers, and in sources (so, basically only in files that are in the `src` directory). Jeffrey has a strong opinion about this, that compilers are free to interpret quote-includes however they wish, and not necessary to start looking in the current folder, so you're not gaining anything from using quote includes, so our approach is to use angle bracket includes, unless you have to use the quoted ones, and, again, we don't have a single quote-style-include in our public headers.

Reply:
Also, I mentioned this above, we'd
```
#include <az_iot_hub_client.h>
#include <az_iot_mqtt.h>
#include <az_span.h>
#include <az_result.h>

#include <time.h>

#include <_az_cfg_prefix.h>
```

Reply:
Not relevant for the next commit as we've coalesced all .h into 2 files for the two services we support.
Makes sense and I'll keep that in mind.